src/redux/reducers/vendorReducer.js

```diff
@@ -0,0 +1,31 @@
+import {REGISTER_BUSINESS_FAIL,REGISTER_BUSINESS_SUCCESS} from '../actions/type';
+
+const initialState = {
```

Comment:
@ritwizsinha You don't need all this inside the initial state. Just have an msg:''.

Reply:
done